packages/calcite-components/src/components/table-cell/table-cell.scss

```diff
@@ -24,10 +34,16 @@
 }
 
 td {
-  @apply text-start align-middle text-color-1 whitespace-normal;
-  background: var(--calcite-internal-table-cell-background);
+  @apply text-start align-middle  whitespace-normal;
+  color: var(--calcite-table-cell-text-color, var(--calcite-color-text-1));
+
+  background-color: var(
+    --calcite-internal-table-cell-background-color,
+    var(--calcite-table-cell-background-color, transparent)
+  );
   font-size: var(--calcite-internal-table-cell-font-size);
-  border-inline-end: 1px solid var(--calcite-color-border-3);
+  border-inline-end: 1px solid
```

Comment:
nitpick: these could use the 1px border var. Applies everywhere border thickness is used.